lib/logstash/inputs/tcp.rb

```diff
@@ -22,12 +21,6 @@ class LogStash::Inputs::Tcp < LogStash::Inputs::Base
   # When mode is `client`, the port to connect to.
   config :port, :validate => :number, :required => true
 
-  # The 'read' timeout in seconds. If a particular tcp connection is idle for
-  # more than this timeout period, we will assume it is dead and close it.
-  #
-  # If you never want to timeout, use -1.
-  config :data_timeout, :validate => :number, :default => -1
```

Comment:
agreed, should probably be set to deprecated, specially in this case where it says ```If a particular tcp connection is idle```, I see people still using it.

what do you think?